src/test/java/com/amazon/opendistroforelasticsearch/security/HttpIntegrationTests.java

```diff
@@ -705,6 +714,67 @@ public void testSslOnlyMode() throws Exception {
 
         res = rh.executeGetRequest("/_search");
         Assert.assertEquals(HttpStatus.SC_OK, res.getStatusCode());
+
+        if (dualModeEnabled) {
+            res = rh.executeGetRequest("/_opendistro/_security/ssl_dual_mode");
```

Comment:
IMO, it should be part of API testing, not HttpIntegrationTests.

Reply:
Moved these tests to a separate file